paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreen.kt

```diff
@@ -35,6 +57,21 @@ internal fun UpdateCardScreenBody(
             style = MaterialTheme.typography.h2
         )
 
+        StripeThemeForLink {
+            CardDetailsEditUI(
+                editCardDetailsInteractor = interactor,
+            )
+        }
+
+        if (isDefault) {
+            Text(
+                modifier = Modifier.padding(top = 8.dp),
+                text = stringResource(R.string.stripe_link_update_card_default_card),
+                style = MaterialTheme.typography.subtitle2,
```

Comment:
Fine for now. I'll revisit this once I get https://github.com/stripe/stripe-android/pull/10716 ready.